### PR TITLE
Release 1.6.17

### PR DESCRIPTION
**Pull request by stevemessick**
I made a mistake during the release process for 1.6.16.

### CODE REVIEW

**Review by rosbo — COMMENTED** [review]
Looks like 1.6.16 was never released to Pypi so you should be good to retry: https://pypi.org/project/kaggle/#history

**Comment by stevemessick** [comment]
No such luck. The test release fails.
```bash
Step #4 - "release-pypitest": WARNING  Error during upload. Retry with the --verbose option for more details. 
Step #4 - "release-pypitest": ERROR    HTTPError: 400 Bad Request from https://test.pypi.org/legacy/          
Step #4 - "release-pypitest":          File already exists. See https://test.pypi.org/help/#file-name-reuse   
Step #4 - "release-pypitest":          for more information.                                                  
```

**Review by rosbo — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 Changelog
 ====
+### 1.6.17
+
+* No changes; release 1.6.15 did not complete.
```
Comment (rosbo) [inline CHANGELOG.md:5]: release 1.6.16 did not complete?

Comment (stevemessick) [reply CHANGELOG.md:5]: Thanks, I'll fix that.

**Pull request merged**